ImperatorToCK3/Source/CK3Outputter/outProvinces.cpp

```diff
@@ -19,7 +19,7 @@ void CK3::outputHistoryProvinces(const std::string& outputModName, const std::ma
 	//create province mapping dummy
 	unsigned char bom[] = { 0xEF,0xBB,0xBF };
 	std::ofstream dummy("output/" + outputModName + "/history/province_mapping/dummy.txt");
-	dummy.write(reinterpret_cast<char*>(bom), sizeof(bom)); // to make the file utf8-bom
+	dummy.write(reinterpret_cast<char*>(bom), sizeof bom); // to make the file utf8-bom
```

Comment:
Didn't you add BOM to commons or am I misremembering?
If you did could you just dummy << commonItems::BOM or something?

Reply:
I did, just didn't go through the already added outputters to add it. Thanks for spotting this!

Reply:
Yeah, it's just output << common items::utf8BOM;